### PR TITLE
Seperation of Settings UI pages into Flask Blueprint and update to devtool with Flask8 check

### PR DESCRIPTION
**Pull request by microtechno9000**
# Description

Two changes
1. Separation of the settings pages into their own Flask Blueprint folder and structure, to start separating out the massive routes file
2. Addition of flask8 quality check to the dev tools, with the check copied from the github scripts

Fixes #
#656 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Not tested with a dvd/bluray; however, no change to the ripper code has been made. Change tested on multiple systems (with and without drives), and all forms/input tested.

- [ ] Ubuntu 18.04
- [x] Ubuntu 20.04
- [ ] Debian 10
- [ ] Debian 11
- [x] Docker
- [ ] Other (Please state here)

# Checklist:

- [x] **I have submitted this PR against the `v2_devel` branch (Unless the main branch has a security issue)**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

### CODE REVIEW

**Comment by shitwolfymakes** [comment]
DVD Rip successful, see log attached

[POINT_BREAK.log](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/10416236/POINT_BREAK.log)

File: arm/migrations/versions/95623e8c5d58_merge_add_notification_and_create_sys_.py [hunk 1]
```diff
@@ -18,8 +18,9 @@
 
 def upgrade():
```
Comment (shitwolfymakes) [inline arm/migrations/versions/95623e8c5d58_merge_add_notification_and_create_sys_.py:19]: Add a TODO comment explaining what this function will eventually do

File: arm/migrations/versions/95623e8c5d58_merge_add_notification_and_create_sys_.py [hunk 2]
```diff
@@ -18,8 +18,9 @@
...
     pass
-    #nothing to upgrade or downgrade, merge two forked database migrations into one
+    # nothing to upgrade or downgrade, merge two forked database migrations into one
+
 
 def downgrade():
```
Comment (shitwolfymakes) [inline arm/migrations/versions/95623e8c5d58_merge_add_notification_and_create_sys_.py:24]: Add a TODO comment explaining what this function will eventually do

File: devtools/armdevtools.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 import armui
 import log
 import database
+import qacheck
```
Comment (shitwolfymakes) [inline devtools/armdevtools.py:11]: Don't forget to add this to the requirements.txt

**Comment by shitwolfymakes** [comment]
Blu-ray test completed, see attached log.

[MOULIN_ROUGE.log](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/10416711/MOULIN_ROUGE.log)

File: devtools/armdevtools.py (hunk 3 above)
Comment (microtechno9000) [reply devtools/armdevtools.py:11]: updated requirements with flake8 to support the devtools

**Review by shitwolfymakes — APPROVED** [review]

**Pull request merged**